packages/zoe/src/contracts/callSpread/payoffHandler.js

```diff
@@ -52,15 +50,20 @@ function makePayoffHandler(zcf, seatPromiseKits, collateralSeat) {
   function payoffOptions(quoteAmount) {
     const strike1 = terms.strikePrice1;
     const strike2 = terms.strikePrice2;
-    const { longShare, shortShare } = calculateShares(
+    const { longShare } = calculateShares(
       collateralBrand,
       quoteAmount,
       strike1,
       strike2,
     );
+    const totalCollateral = terms.settlementAmount;
+    // round in favor of the long position
+    const longPortion = ceilMultiplyBy(totalCollateral, longShare);
+    const shortPortion = AmountMath.subtract(totalCollateral, longPortion);
```

Comment:
I like that these are now calculated together rather than having computations in different places that have to line up.